src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -0,0 +1,60 @@
+package de.tum.in.www1.artemis.repository;
+
+import java.time.Instant;
+import java.time.ZonedDateTime;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.User;
+
+/**
+ * Spring Data JPA repository for the user statistics
+ */
+@Repository
+public interface StatisticsRepository extends JpaRepository<User, Long> {
+
+    @Query("select count(distinct u.login) from User u, PersistentAuditEvent p where u.login like p.principal and p.auditEventType = 'AUTHENTICATION_SUCCESS' and u.login not like '%test%' and p.auditEventDate >= :#{#span}")
+    Integer getLoggedInUsers(@Param("span") Instant span);
```

Comment:
As far as I can see all the querys here query for something in certain range (from dateA to dateB), but sometimes you use only one parameter, and sometimes you use two parameters. Could you maybe rewrite the querys to always use two parameters? IMO it makes them easier to understand because I can clearly see: "this gets the logged in users from data A to data B" and the querys become more useful as i can input any timeframe i want

Reply:
Also maybe rename the methods to something more clearer like `countNumberOfLoggedInUsers`, so it is more clear from the method name what the quer does

Reply:
Yes, sorry, like mentioned in the PR description, this is deprecated code which will be refactored after this PR and is just here for semantic purposes.

Reply:
I was also very confused until I read @stefanwaldhauser's comment. If the code is going to be refactored and deprecated it might be a good idea to add a comment above the method and mention that. Just to not confuse others